### ISSUE TITLE
Didn't find class "com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView"

### ISSUE THREAD

**Issue by GreenhairTurtle** [voghDev/PdfViewPager#110]
I am not sure if there is a problem with your gradle

```java
android.view.InflateException: Binary XML file line #13: Binary XML file line #13: Error inflating class com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
    Caused by: android.view.InflateException: Binary XML file line #13: Error inflating class com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
    Caused by: java.lang.ClassNotFoundException: Didn't find class "com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/base.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_dependencies_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_0_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_1_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_2_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_3_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_4_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_5_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_6_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_7_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_8_apk.apk", zip file "/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/lib/arm64, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/base.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_dependencies_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_0_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_1_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_2_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_3_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_4_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_5_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_6_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_7_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_8_apk.apk!/lib/arm64-v8a, /data/app/com.wecardio-LS_fmPDHi-6ezv_hc2quhA==/split_lib_slice_9_apk.apk!/lib/arm64-v8a, /system/lib64, /product/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:613)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:801)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:874)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:835)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at es.voghdev.pdfviewpager.library.adapter.PDFPagerAdapter.instantiateItem(PDFPagerAdapter.java:45)
        at android.support.v4.view.ViewPager.addNewItem(ViewPager.java:1004)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1152)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1086)
        at android.support.v4.view.ViewPager.onMeasure(ViewPager.java:1616)
        at android.view.View.measure(View.java:23480)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6831)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
```

And this problem was solved after I introduced this library myself.

**Comment by voghDev** [comment]
Hello,

thanks for reporting this issue. I'll check it out

**Comment by GreenhairTurtle** [comment]
Ok,thank you

**Comment by saltwater2233** [comment]
Same problem，its just happen in the 1.0.6
when i return 1.0.5，its work

**Comment by voghDev** [comment]
I've been able to reproduce this issue in my phone. As a temporary solution while I discover the root cause, please add SubsamplingScaleImageView as a separate dependency

        implementation 'com.davemorrissey.labs:subsampling-scale-image-view:3.10.0'

I'm sorry about that guys. Will fix it in the next version of the library

**Comment by YuLingRui** [comment]
04-19 09:48:35.226 2951-2951/com.pdfweb E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.pdfweb, PID: 2951
    android.view.InflateException: Binary XML file line #13: Error inflating class com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:757)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:806)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:504)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:414)
        at es.voghdev.pdfviewpager.library.adapter.PDFPagerAdapter.instantiateItem(PDFPagerAdapter.java:45)
        at android.support.v4.view.ViewPager.addNewItem(ViewPager.java:1010)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1158)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1092)
        at android.support.v4.view.ViewPager.onMeasure(ViewPager.java:1622)
        at android.view.View.measure(View.java:17547)
        at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:727)
        at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:463)
        at android.view.View.measure(View.java:17547)
        at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:727)
        at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:463)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5535)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
        at android.support.v7.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:143)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5535)
        at android.support.v7.widget.ActionBarOverlayLayout.onMeasure(ActionBarOverlayLayout.java:401)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5535)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5535)
        at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1436)
        at android.widget.LinearLayout.measureVertical(LinearLayout.java:722)
        at android.widget.LinearLayout.onMeasure(LinearLayout.java:613)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5535)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:436)
        at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2615)
        at android.view.View.measure(View.java:17547)
        at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2039)
        at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1197)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1403)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1085)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5909)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
        at android.view.Choreographer.doCallbacks(Choreographer.java:580)
        at android.view.Choreographer.doFrame(Choreographer.java:550)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5293)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)

**Comment by nikhilchhabra** [comment]
Hi, it is still not working.. Please release a new version with the fix . Thanks :)

**Comment by android-bash** [comment]
Yes still not working. Please fix it asap. Thank you for the library.

**Comment by YuLingRui** [comment]
是我的pdf文件太大的问题吗？？？ 我把路径放到你的demo中也不行




------------------ 原始邮件 ------------------
发件人: "android-bash"<notifications@github.com>; 
发送时间: 2019年4月23日(星期二) 下午3:16
收件人: "voghDev/PdfViewPager"<PdfViewPager@noreply.github.com>; 
抄送: "安东尼"<1307444320@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [voghDev/PdfViewPager] Didn't find class"com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView" (#110)




Yes still not working. Please fix it asap. Thank you for the library.

**Comment by voghDev** [comment]
Fixed in 1.1.0, together with the migration to androidx. Thanks guys for reporting
